docs/user_guide.md

```diff
@@ -719,6 +721,200 @@ If you are a project admin, you can disable a Robot Account by clicking "Disable
+- You set the retention policy to retain the 10 most recently pulled images in each repository.
+- You set the tag filter to `**`, meaning that all tags in the repository are included.
+
+In this example the rule retains the 10 most recently pulled images in repository A, and all of the images in repositories B to E, retaining a total of 34 image tags in the project. In other words, the rule does not treat all of the images in repositories A to E as a single pool from which to choose the 10 most recent images. So, even if the 11th to 100th tags in repository A have been pulled more recently than any of the tags in repositories B to E, all of the tags in repositories B to E are retained, because each of those repositories has fewer than 10 tags.
+
+#### Example 2
+
+This example uses the same project and repositories as example 1, but sets the retention policy to retain the images in each repository that have been pulled in the last 7 days.
+
+In this case, all of the images in repository A are retained because they have been pulled in the last 7 days. None of the images in repositories B to E are retained, because none of them has been pulled in the last week. In this example, 100 images are retained, as opposed to 34 images in example 1.
+
+### Combining Rules on a Respository
+
+You can define up to 15 rules per project. You can apply multiple rules to a repository or set of repositories. When you apply multiple rules to a repository, they are applied with `OR` logic rather than with `AND` logic. In this way, there is no prioritization of application of the rules on a given repository. 
+
+#### Example 3
+
+This example uses the same project and repositories as examples 1 and 2, but sets two rules:
+
+- Rule 1: Retain all of the images in each repository that have been pulled in the last 7 days.
+- Rule 2: Retain a maximum number of 10 images in each repository.
+
+In this case, repository A only retains the 10 most recent images, even though images 11 to 100 have been pulled in the last week. Repositories B to E retain no images, because none of those images have been pulled in the last week. In this example, 10 images are retained, as opposed to 34 images in example 1 and 100 in example 2.
+
+
+#### Example 4
+
+This example uses a different repository to the previous examples.
+
+- You have a repository that has 12 tags:
+
+  |Production|Release Candidate|Release|
+  |---|---|---|
+  |`2.1-your_repo-prod`|`2.1-your_repo-rc`|`2.1-your_repo-release`|
+  |`2.2-your_repo-prod`|`2.2-your_repo-rc`|`2.2-your_repo-release`|
+  |`3.1-your_repo-prod`|`3.1-your_repo-rc`|`3.1-your_repo-release`|
+  |`4.4-your_repo-prod`|`4.4-your_repo-rc`|`4.4-your_repo-release`| 
+
+- You define three tag retention rules on this repository:
+  - Retain the 10 most recently pushed image tags that start with `2`.
+  - Retain the 10 most recently pushed image tags that end with `-prod`.
+  - Exclude the image tag `2.1-your_repo-prod` from the list of tags to retain.

```

Comment:
This rule is not clear as it's not defined with the retention rule pattern (shown in the UI, not the backend meanings).

Reply:
@steven-zou I'm not sure what you mean here. Can you please explain a bit more?

Reply:
I mean this rule statement`Exclude the image tag `2.1-your_repo-prod` from the list of tags to retain.` is not following the existing retention rule pattern as it misses some parts, it might be something like:

`Retain always tags that excludes 2.1-your_repo-prod`

Reply:
@steven-zou, so if in the UI I select tag *excluding* and enter `2.1-your_repo-prod` does that mean that all tags except for `2.1-your_repo-prod` are retained? Or that only `2.1-your_repo-prod` is retained?

Reply:
@stuclem  He's not referring to the behavior.  He's referring to the format, that all policies start with 'retain ...', so 'Exclude the image tag `2.1-your_repo-prod` from the list of tags to retain.' should perhaps be better stated as 'Retain all tags excluding ...'   To answer your question in the last comment,  if in the UI you select tag excluding and enter 2.1-your_repo-prod, then all tags except 2.1-your_repo-prod are retained.

@steven-zou I don't think your way is necessarily better because this is a user guide, it should follow natural speech pattern, but maybe ok for consistency sake. 

You guys can comment on which is actually better. I just wanted to clear up some misunderstanding.